zapisy/apps/schedule/views.py

```diff
@@ -337,6 +337,50 @@ def get_queryset(self):
         return queryset
 
 
+def freedays(request):
+    """Returns list of freedays between 'start' and 'end' dates.
+
+    Both dates has to be given in 'input_date_format'. Date is returned in 'output_date_format'.
+    Returns 404 if one or both paramas are missing.
```

Comment:
```suggestion
    Both dates have to be given in 'input_date_format'. Date is returned in 'output_date_format'.
    Raises 404 if one or both params are missing.
```
A co do tej "zwracanej daty" – czy to jest prawda? Data, a właściwie daty, specjalnych dni, są zwracane w akurat takim formacie, jaki wypadnie z ~~maszyny losującej~~ bazy danych (via Django). Ten `output_date_format` wygląda na raczej dostosowany do tego, żeby zapytanie do bazy było dobrze napisane (co z drugiej strony oczywiście oznacza zapewne dokładnie ten sam format, ale zależność przyczynowo-skutkowa jest inna, niż sugeruje sformułowanie z komentarza).

Trzeba to jakoś przeformułować, bo obecnie to sugeruje, że zmieniając `output_date_format` uzyskamy inne sensowne zachowanie tej funkcji, a zdaje się, że możemy ją w ten sposób tylko zepsuć.